docs/source/plugins/03_plugin_structure.rst

```diff
@@ -4,48 +4,41 @@ Plugins Structure
 =================
 
 As mentioned in :ref:`quick-start-section`, the |sdk| package has some utilities to help in the process to create
-a new plugin project and to create a plugin file.
+a plugin file.
 
-At any moment, it is possible to invoke the ``help`` command to list all commands available.
+First, for the creation of the plugin you can use the CLI automatic generator
```

Comment:
We tend in Portuguese to be too verbose when compared to English, I suggest this which is short and to the point:

```suggestion
First, to create the plugin use :program:`new`:
```